app/jobs/scheduled/topic_timer_enqueuer.rb

```diff
@@ -12,13 +12,15 @@ class TopicTimerEnqueuer < ::Jobs::Scheduled
     every 1.minute
 
     def execute(_args = nil)
-      timers = TopicTimer.pending_timers
-
-      timers.find_each do |timer|
+      TopicTimer.pending_timers.find_each do |timer|
 
         # the typed job may not enqueue if it has already
         # been scheduled with enqueue_at
-        timer.enqueue_typed_job
+        begin
+          timer.enqueue_typed_job
+        rescue => err
+          Rails.logger.error("Error when attempting to enqueue topic timer job for timer #{timer.id}: #{err.class} #{err.message}: #{err.backtrace.join("\n")}")
```

Comment:
Let's use `Discourse.warn_exception` so that the error is logster friendly.

Reply:
@tgxworld I didn't know about that, thanks!